portfolio/src/main/webapp/style.css

```diff
@@ -1,3 +1,22 @@
+a {
+    text-decoration: none;
+}
+
+button {
+    background: whitesmoke;
+    width: 70px;
+    height: 70px;
+    border: thin solid rgba(0, 0, 0, 0.726);
+}
```

Comment:
Put declarations in alphabetical order.
Also below
https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order